src/sagas/ideas.test.js

```diff
@@ -0,0 +1,34 @@
+import { put, takeLatest } from 'redux-saga/effects';
+import { fetchIdeas, catchError, getIdeas, ideasActionWatcher } from './ideas'
+
+describe('ideas saga', () => {
+    test('should dispatch action "FETCH:IDEAS" ', () => {
+        const generator = ideasActionWatcher();
+        expect(generator.next().value).toEqual(takeLatest('FETCH:IDEAS', getIdeas));
+        expect(generator.next().done).toBeTruthy();
```

Comment:
You should avoid more than one assertion per test case. If it fails, you'll only see _that_ it failed on CI (not sure with local), so it's not clear which of the two. You circumvent that problem, by only ever having one assertion per test-case.